src/util/index.ts

```diff
@@ -0,0 +1,6 @@
+import { checkMinLength } from "./checkMinLength";
```

Comment:
Please use here another approach
You need to make it like that:
```export * from  './checkMinLength'```
And other

Reply:
Done.